libbeat/cmd/instance/beat.go

```diff
@@ -695,10 +690,9 @@ func (b *Beat) loadDashboards(ctx context.Context, force bool) error {
 			withMigration = sub.Enabled()
 		}
 
-		// init kibana config object
-		kibanaConfig := b.Config.Kibana
-		if kibanaConfig == nil {
-			kibanaConfig = common.NewConfig()
+		kibanaConfig, err := initKibanaConfig(b.Config)
```

Comment:
Can you add a comment here about the order of it to make sure it does not get moved again in the future by accident?

Reply:
Good idea! Done!